src/knex-tables.d.ts

```diff
@@ -148,10 +150,75 @@ declare module "knex/types/tables" {
     id: number;
     onerep_profile_id: number;
     onerep_scan_id: number;
-    onerep_scan_results: ScanResult;
+    onerep_scan_reason: Scan["reason"];
     created_at: Date;
     updated_at: Date;
```

Comment:
I wonder if it's still necessary to have `updated_at` as a col. Seems like an append-only table now

Reply:
Hmm, I'm not familiar enough with Scans to be able to tell with certainty. Don't scans go through several states (i.e. [there's](https://docs.onerep.com/#operation/getScan) a `status` property of which one possible value is at least `in_progress`) - or are we only inserting new ScanResults as they come in, and leave the Scan as-is, since we don't seem to be storing that status? /cc @rhelmer

Reply:
We might update these as a part of the webhook events we will be receiving. Right now it's kind of hazy as to what those events will be like (whether it's an update or a new insert). 
So I guess we can leave it the way it is for now.

Reply:
Ah OK, I'll leave that for a future followup then, thanks!